services/graylog/docker-compose.letsencrypt.yml

```diff
@@ -0,0 +1,7 @@
+version: '3.7'
+services:
+  graylog:
+    deploy:
+      labels:
+        - traefik.http.routers.graylog.rule=Host(`${MONITORING_DOMAIN}`) && PathPrefix(`/graylog`)
+        - traefik.http.routers.graylog.tls.certresolver=lehttpchallenge
```

Comment:
wow. thats enough?

Reply:
yeah well, together with the docker-compose.yml file yes. so about 5-6 labels + the initial traefik configuration.